### PR TITLE
Add stubs for Acl

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
AclProviderInterface.stub contains a parse error: https://phpstan.org/r/5b9d1718-66c8-40e6-8c0b-d9363de991f7

File: stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ <?php
```
Comment (mhujer) [inline stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub:1]: There is a space in the beginning of the line 😄 

```suggestion
<?php
```

Comment (ondrejmirtes) [reply stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub:1]: With PHPStan's dev-master, the experience is a bit nicer:

```
------ --------------------------------------------------------------------------------------------
  Line   stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub
 ------ --------------------------------------------------------------------------------------------
  3      Namespace declaration statement has to be the very first statement in the script on line 3
  3      Namespace declaration statement has to be the very first statement in the script on line 3
  3      Namespace declaration statement has to be the very first statement in the script on line 3
  3      Namespace declaration statement has to be the very first statement in the script on line 3
 ------ --------------------------------------------------------------------------------------------
```

Comment (VincentLanglet) [reply stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub]: Whoah, nice catch

**Comment by VincentLanglet** [comment]
I'm not sure the stub is working as expected. @ondrejmirtes 

The following PR is failing:
https://github.com/sonata-project/SonataAdminBundle/pull/7095

```
    /**
     * @var MutableAclProviderInterface
     */
    protected $aclProvider;

    public function getObjectAcl(ObjectIdentityInterface $objectIdentity)
    {
        try {
            return $this->aclProvider->findAcl($objectIdentity);
        } catch (AclNotFoundException $e) {
            return null;
        }
    }
```
is throwing an error
```
Method                                                                 
         Sonata\AdminBundle\Security\Handler\AclSecurityHandler::getObjectAcl(  
         ) should return                                                        
         Symfony\Component\Security\Acl\Model\MutableAclInterface|null but      
         returns Symfony\Component\Security\Acl\Model\AclInterface.
```
even if the stub is
```
    /**
     * @phpstan-param array<SecurityIdentityInterface> $sids
     * @phpstan-return MutableAclInterface
     */
    public function findAcl(ObjectIdentityInterface $oid, array $sids = []);
```

**Comment by ondrejmirtes** [comment]
@VincentLanglet Please open an issue.

**Comment by VincentLanglet** [comment]
Done https://github.com/phpstan/phpstan-symfony/issues/149

**Pull request merged**